framework/Optimizers/SimulatedAnnealing.py

```diff
@@ -587,37 +593,35 @@ def _applyFunctionalConstraints(self, suggested, previous):
   # Utility Methods #
   ###########
   def _temperature(self, fraction):
-    #return max(0.01,min(1,1-fraction))
+    """
+    A utility function to compute the initial temperaturte
+    currently it is just a function of how far in the process are we
+    @ In, fraction, float, the current iteration devided by the iteration limit i.e., $\frac{iter}{Limit}$
+    @ Out, T0, float, initial temperature, i.e., $T0 = max(0.01,1-fraction) $
+    """
     return max(0.01,1-fraction)
```

Comment:
since you do not have a variable as output, our standard requires to specify Out as the function name:


```
@ Out, _temperature, float, initial temperature, i.e., $T0 = max(0.01,1-fraction) $

```